relay-system/src/service.rs

```diff
@@ -899,6 +899,11 @@ pub fn channel<I: Interface>(name: &'static str) -> (Addr<I>, Receiver<I>) {
     (addr, receiver)
 }
 
+/// A state interface for [services](`Service`).
+pub trait State {}
```

Comment:
Is this trait intended to be expanded later? Currently I don't think this accomplishes anything.

Reply:
Nope, it was meant to be used as some sort of code documentation which clarifies when a type can be used as a state but it serves no purpose besides that. I don't know if it's common in Rust to do this.

Reply:
In that case I would just document the associated type directly.